src/translator/definitions.h

```diff
@@ -15,6 +15,13 @@ typedef std::vector<Segment> Segments;
 /// Shortcut to AlignedVector<char> for byte arrays
 typedef AlignedVector<char> AlignedMemory;
 
+struct MemoryBundle {
+  AlignedMemory model;
+  AlignedMemory shortlist;
+  std::vector<std::shared_ptr<AlignedMemory>> vocabs;
+  AlignedMemory ssplitPrefixFile;
+};
```

Comment:
Move documentation from Service class here. All the aligned to X, what the vocab represents bits.